netplan_cli/cli/ovs.py

```diff
@@ -20,10 +20,9 @@
 import subprocess
 import re
 
-from .utils import systemctl_is_active
-
-OPENVSWITCH_OVS_VSCTL = '/usr/bin/ovs-vsctl'
-OPENVSWITCH_OVSDB_SERVER_UNIT = 'ovsdb-server.service'
+OPENVSWITCH_OVS_VSCTL = (
```

Comment:
thought: We also have some NetworkManager.snap handling in `cli/utils.py:is_nm_snap_enabled()`, which handles a similar situation a bit differently (checking for the snap related systemd service). Both have in common that the snap path gets a higher priority.

My initial idea was to bring the OVS handling in line with the NM handling, using a similar logic. But OTOH, we also have the changes in `src/openvswitch.c` (that we don't have for NM.snap), mimicking the path-fallback logic implemented here. So it might be better to keep the OVS-VSCL handling in-line between .c and .py and rather adopt/refactor the NM.snap handling at some point in the future.